ibis/expr/api.py

```diff
@@ -1523,6 +1523,10 @@ def convert_base(arg, from_base, to_base):
 }
 
 
+bit_and = _agg_function('bit_and', ops.BitAnd, True)
```

Comment:
can you add doc-strings to these, might be more clear what they do. Also add a link to the docs (bigquery / sql are fine)

Reply:
I updated the docstrings for the operators and pulled the logic from `_unary_op` that copies the docstring from the class into the `_agg_function` definition.

https://github.com/ibis-project/ibis/blob/f22e79f18449b99fb60552779ccaccd1aee74514/ibis/expr/api.py#L530